Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -842,7 +871,12 @@ static int SavingThrow(DFCareer.EffectFlags effectFlags, DaggerfallEntity target
             if (toleranceFlags == DFCareer.Tolerance.LowTolerance)
                 savingThrow = 75;
 
-            savingThrow += biographyMod;
+            savingThrow += biographyMod + modifier;
```

Comment:
This actually answers my question in the other commit. I'm sorry, I hadn't looked through the code when I posted that. This looks most useful thank you!